public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -11,17 +11,33 @@ const TopSection: FC<any> = () => {
     text: 'Search',
     icon: 'search',
   };
-
+  const dbaasLink = {
+    id: 'dbaas',
+    icon: 'database',
+    text: 'DBaaS',
+    url: '/graph/d/pmm-dbaas/pmm-dbaas',
+  };
   const onOpenSearch = () => {
     getLocationSrv().update({ query: { search: 'open' }, partial: true });
   };
+  const [showDBaaS, setShowDBaaS] = useState(false);
+
+  // TODO: once DBaaS is enabled by default move it to config file (api/index.go)
+  useEffect(() => {
+    if (config.bootData.user.isSignedIn) {
+      getBackendSrv()
+        .post('http://localhost/v1/Settings/Get')
+        .then(({ settings }) => setShowDBaaS(settings.dbaas_enabled));
```

Comment:
Use async/await here please

Reply:
That would need an extra step of creating an IIFE just because of that. And I see that they use this syntax a lot throughout the project when they use getBackendSrv, so I'm wondering if it's really worth it to go that extra step in this case.

Reply:
It forces you to create another separate function (not an just an IIFE), which woulnd't look bad, quite the opposite :)